gaming/servers/create_config.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package servers
+
+// [START cloud_game_servers_config_create]
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	gaming "cloud.google.com/go/gaming/apiv1beta"
+	gamingpb "google.golang.org/genproto/googleapis/cloud/gaming/v1beta"
+)
+
+const fleet = `
```

Comment:
Perhaps include a comment explaining the format of this?

Reply:
done.